src/Jackalope/ObjectManager.php

```diff
@@ -1806,4 +1806,22 @@ public function purgeDisappearedNode($absPath, $keepChanges)
         // objectsByPath and the calling parent node can forget it
         return true;
     }
+
+    /**
+     * Updates a single node via the transport layer if the node is modified
+     * @param NodeInterface $node The node to update
+     */
+    private function updateNode($node)
+    {
+        /** @var $node Node */
```

Comment:
lets put the typehint into the method. then we can drop this comment, and also the weird instanceof check below. its only a sanity check - the typehint is better for that.